dev/PushNotifications/PushNotificationsLongRunningTask.StartupTask/winmain.cpp

```diff
@@ -0,0 +1,23 @@
+﻿#include <windows.h>
+#include "pch.h"
+
+#include "../PushNotifications-Constants.h"
+
+#include <NotificationsLongRunningProcess_h.h>
+#include <wil/resource.h>
+
+int WINAPI WinMain(HINSTANCE /*hInstance*/, HINSTANCE /*hPrevInstance*/, PSTR /*lpCmdLine*/, int /*nCmdShow*/)
+{
+    RETURN_IF_FAILED(::CoInitializeEx(nullptr, COINITBASE_MULTITHREADED));
+
+    try
+    {
+        wil::com_ptr<INotificationsLongRunningPlatform> longRunningProcessPlatform
+            { wil::CoCreateInstance<NotificationsLongRunningPlatform, INotificationsLongRunningPlatform>(CLSCTX_LOCAL_SERVER) };
+    }
+    CATCH_LOG()
+
+    CoUninitialize();
```

Comment:
Leak

Reply:
The CoCreateInstance call is in a try-catch, how is this a leak?